mcsema/BC/Util.cpp

```diff
@@ -54,6 +54,26 @@ llvm::FunctionType *LiftedFunctionType(void) {
   return func_type;
 }
 
+// Translate `ea` into an LLVM value that is an address that points into the
+// external variable
+llvm::Constant *LiftExternalEA(const NativeExternalVariable *cfg_ext,
+                               uint64_t ea) {
+  LOG(INFO) << "external variable is " << std::hex << cfg_ext->ea << " ea is "
+            << std::hex << ea << "\n";
```

Comment:
Always reset the display to `std::dec` at the end of a `LOG(INFO)`, and the log lines don't need a terminating new line.